build/package.json

```diff
@@ -41,7 +41,7 @@
     "minimist": "^1.2.0",
     "request": "^2.85.0",
     "tslint": "^5.9.1",
-    "service-downloader": "github:anthonydresser/service-downloader#0.1.5",
+    "service-downloader": "d:/git/service-downloader",
```

Comment:
This will be changed to actual version of "github:anthonydresser/service-downloader#0.1.7" prior to checkin. 
This merge is dependent on PR: https://github.com/anthonydresser/service-downloader/pull/11